### ISSUE TITLE
Issues with AND and OR Filters

### ISSUE THREAD

**Issue by jmmcatee** [jmckaskill/goldap#4]
Looks like there is an encoding issue when using the AND or OR filters for LDAP filtering. Any experience with this yourself?

**Comment by akshaybahetii** [comment]
Yes, I came across some encoding issue with AND and OR filters. @jmmcatee  Did you find a fix for it ? Or did you use some other go ldap library.
